packages/storybook/stories/MessageThread/MessageThread.stories.tsx

```diff
@@ -192,6 +197,11 @@ const Docs: () => JSX.Element = () => {
         checkout the details about these components
         [here](https://microsoft.github.io/fluentui-contrib/react-chat/?path=/story/chat--default).
       </Description>
+      <Description>
+        The MessageThread component supports lazy loading for the rich text editor used for editing messages. This means
```

Comment:
Can we include a link to sideEffects doc?

https://webpack.js.org/guides/tree-shaking/#mark-the-file-as-side-effect-free

Reply:
Updated!